bonsai/bonsai.go

```diff
@@ -78,14 +93,46 @@ func NewBaseAsset(name string) (*BaseAsset, error) {
 	return &bAsset, nil
 }
 
+// BonsaiVersion returns the Bonsai version for a given requested version, and
+// fallbacks to the latest version if the requested version is nil. An error is
+// returned if the requested version does not exist
+func (b *Asset) BonsaiVersion(version *goversion.Version) (*goversion.Version, error) {
+	if version == nil {
+		// No version was requested, therefore return the latest version
+		v := b.LatestVersion()
+		if v == nil {
+			return nil, fmt.Errorf("asset %q does not have any available versions", b.Name)
+		}
+
+		fmt.Println("no version specified, using latest:", v.Original())
```

Comment:
Should probably be removed?

Reply:
It was actually already present, which produces the following:

```
$ sensuctl asset add nixwiz/sensu-go-fatigue-check-filter
no version specified, using latest: 0.4.0
fetching bonsai asset: nixwiz/sensu-go-fatigue-check-filter:0.4.0
added asset: nixwiz/sensu-go-fatigue-check-filter:0.4.0
```

Reply:
Or maybe moved to somewhere less library-ish?